src/test/arg/basics.spec.ts

```diff
@@ -91,25 +91,60 @@ describe('Arguments with @Arg', () => {
   });
 
   it('Will allow registering argument at runtime', () => {
+    @ObjectType()
```

Comment:
This test should keep focusing only on registering args at runtime. Having default values tested here pollutes this  test

Reply:
yeah I agree, I haven't actually changed this test at all, I have no idea why github has decided to render the diff like this, my own tools don't. Though it does look like something autoformatted the parameter block (line 97), maybe that's it.